src/main/target/common_pre.h

```diff
@@ -339,6 +339,12 @@ extern uint8_t _dmaram_end__;
 
 #endif // !defined(CLOUD_BUILD)
 
+#if defined(USE_64_LED)
+    #define LED_MAX_STRIP_LENGTH 64
+#else
+    #define LED_MAX_STRIP_LENGTH 32
+#endif
```

Comment:
We should wrap the code:

```suggestion
#if defined(USE_LED_STRIP)
#define USE_LED_STRIP_STATUS_MODE
#if defined(USE_64_LED)
    #define LED_MAX_STRIP_LENGTH 64
#else
    #define LED_MAX_STRIP_LENGTH 32
#endif
#endif
```
Then the following should be removed on line 438 (444)

    #define USE_LED_STRIP_STATUS_MODE

should be removed from line 438 (444)

Reply:
Suggestion won't work.
USE_LED_STRIP is a flag that indicates if there is a LED function at all or not regardless if it's 64 or 32

Reply:
Yes. Assuming here cloud build option `LED` would `#define USE_LEDSTRIP`